Runar/NotificationContentExtension/NotificationViewController.swift

```diff
@@ -19,15 +19,12 @@ class NotificationViewController: UIViewController, UNNotificationContentExtensi
     
     func didReceive(_ notification: UNNotification) {
         
-        //MARK: - Notification Actions
+        let attachment = notification.request.content.attachments.first(where: {$0.identifier == "attachment"})
```

Comment:
please, change for this

let bannerAttachment = notification.request.content.attachments.first {
        $0.identifier == "attachment"
}